lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/v2x/etsi/DenmContent.java

```diff
@@ -94,6 +93,12 @@ public DenmContent(final long time, final GeoPoint senderPosition, final String
         this.eventLocation = eventLocation;
         this.eventArea = eventArea;
         this.extendedContainer = extendedContainer;
+
+        if (senderPosition != null) {
+            this.senderPosition = senderPosition;
+        } else {
+            this.senderPosition = null;
+        }
```

Comment:
This is exactly the same as `this.senderPosition = senderPosition;`. you can keep this constructor as it is